tests/conftest.py

```diff
@@ -74,23 +64,58 @@ def auth_state(MockAuthClient, config, monkeypatch) -> AuthState:
     return auth_state
 
 
-@pytest.fixture
-def duplicate_auth_state(auth_state: AuthState, config) -> AuthState:
-    """This fixture provides an AuthState type object that has the same token value as the
-    auth_state fixture above. This means that caching should make access to either of the
-    returned auth_state objects return values from the same cache as the root key is the
-    token value.
+@pytest.fixture(scope="session", autouse=True)
+def register_api_fixtures():
+    for yaml_file in (pathlib.Path(__file__).parent / "api-fixtures").rglob("*.yaml"):
+        response_set = yaml.safe_load(yaml_file.read_text())
+        register_response_set(yaml_file.stem, response_set)
```

Comment:
I hope this fixture was as much of a delight to write as it is to read! ⭐

Reply:
It sure was! :partying_face: